app/serializers/game_serializer.rb

```diff
@@ -0,0 +1,5 @@
+class GameSerializer < ActiveModel::Serializer
```

Comment:
über nitpick :tm:: some classes have spaces after the first line and before the last `end`, and some don't. Which style do you prefer?